snake-kmp-benchmarks/src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/benchmark/DumpBenchmark.kt

```diff
@@ -0,0 +1,47 @@
+package it.krzeminski.snakeyaml.engine.kmp.benchmark
+
+import it.krzeminski.snakeyaml.engine.kmp.api.Dump
+import it.krzeminski.snakeyaml.engine.kmp.api.DumpSettings
+import it.krzeminski.snakeyaml.engine.kmp.api.StreamDataWriter
+import kotlinx.benchmark.*
+import okio.Buffer
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(BenchmarkTimeUnit.MICROSECONDS)
+class DumpBenchmark {
+    private val dumper = Dump(
+        DumpSettings.builder()
+            .build(),
+    )
+
+    private val streamDataWriter = object : StreamDataWriter {
+        val buffer = Buffer()
+        override fun write(str: String) {
+            buffer.writeUtf8(str)
+        }
+
+        override fun write(str: String, off: Int, len: Int) {
+            buffer.writeUtf8(str, off, len)
+        }
+
+    }
+
+    private val objectToDump: Any = linkedMapOf(
```

Comment:
I'm a little worried that this structure is too simple to give meaningful results in the benchmark, but it's certainly better than nothing. Thanks for adding it!